docs/make.jl

```diff
@@ -3,10 +3,9 @@ push!(LOAD_PATH, "../src/")
 
 using Documenter, ADRIA
 
-
-makedocs(
+makedocs(;
```

Comment:
This is weird, not sure why the semicolon on opening parenthesis.

Reply:
If the only arguments are keyword arguments, this is clear to me:

```julia
function blah(;
    foo=100
)
```

And is preferred over some other weirdness

```julia
function blah(
    ; foo=100
)
```